starter-core/src/main/java/io/micronaut/starter/feature/function/awslambda/AwsLambda.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.function.awslambda;
+
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.Project;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.feature.Feature;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookGroovy;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookJava;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookKotlin;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookMicronautRequestHandlerGroovy;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookMicronautRequestHandlerGroovyJunit;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookMicronautRequestHandlerJava;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookMicronautRequestHandlerKotlin;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookMicronautRequestHandlerJavaJunit;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookMicronautRequestHandlerKotlinJunit;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookMicronautRequestHandlerKotlinTest;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookMicronautRequestHandlerSpock;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookSavedGroovy;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookSavedJava;
+import io.micronaut.starter.feature.function.awslambda.template.awsLambdaBookSavedKotlin;
+
+import javax.inject.Singleton;
+
+@Singleton
+public class AwsLambda implements Feature {
```

Comment:
Should implement `DefaultFeature` as AWS Lambda will be the default for `create-function`